deltachat-ios/Controller/AccountSwitchViewController.swift

```diff
@@ -246,27 +246,38 @@ class AccountCell: UITableViewCell {
         return 54
     }
 
-    var isLargeText: Bool {
+    private var isLargeText: Bool {
         return UIFont.preferredFont(forTextStyle: .body).pointSize > 36
     }
 
-    lazy var accountAvatar: InitialsBadge = {
+    private lazy var accountAvatar: InitialsBadge = {
         let avatar = InitialsBadge(size: 37)
         avatar.isAccessibilityElement = false
         return avatar
     }()
 
-    var selectedAccount: Int?
-    var accountId: Int?
+    private var selectedAccount: Int?
+    private var accountId: Int?
 
-    public lazy var stateIndicator: UIImageView = {
+    private lazy var stateIndicator: UIImageView = {
         let view: UIImageView = UIImageView()
         view.translatesAutoresizingMaskIntoConstraints = false
         view.contentMode = .scaleAspectFit
         return view
     }()
 
-    lazy var accountName: UILabel = {
+    private lazy var mutedIndicator: UIImageView = {
+        let view = UIImageView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.widthAnchor.constraint(equalToConstant: 16).isActive = true
+        view.tintColor = DcColors.middleGray
+        view.image = #imageLiteral(resourceName: "volume_off").withRenderingMode(.alwaysTemplate)
```

Comment:
Why `#imageLiteral` and not `UIImage(named:...)`?

Reply:
idk - copy+paste :)

i find the representation in xcode weird as well. maybe create an issue to make that more ios-like and consistent for all places, so that next copy+paste is "better"

Reply:
i changed the thing to UIImage(named:...) and created an issue for general removal: https://github.com/deltachat/deltachat-ios/issues/2248